nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GetHTTP.java

```diff
@@ -426,60 +423,65 @@ public void onTrigger(final ProcessContext context, final ProcessSessionFactory
             if (accept != null) {
                 get.addHeader(HEADER_ACCEPT, accept);
             }
+            // create the http client
+            try ( final CloseableHttpClient client = clientBuilder.build() ) {
+                // NOTE: including this inner try in order to swallow exceptions on close
+                try {
```

Comment:
Holy smokes, this got butchered, it should just be an indentation of that try block